packages/codemods/src/codeMods/utilities/imports.ts

```diff
@@ -22,13 +22,17 @@ export function renameImport(file: SourceFile, originalImport: string, renamedIm
  */
 export function getImportsByPath(file: SourceFile, pathOrRegex: string | RegExp): ImportDeclaration[] {
   let imps: ImportDeclaration[] = [];
-  if (typeof pathOrRegex === 'string') {
+  if (typeof pathOrRegex === 'string' && !stringIsRegex(pathOrRegex)) {
     imps = file.getImportDeclarations().filter(cond => {
       return cond.getModuleSpecifierValue() === pathOrRegex;
     });
   } else {
+    if (typeof pathOrRegex === 'string' && stringIsRegex(pathOrRegex)) {
```

Comment:
After thinking about this a bit, I think that we shouldn't convert the string to regex. At least not yet. Instead there should be another option in the json that is just regex so that we can be sure it's always parsed in the right way.

Reply:
That makes sense -- I didn't like how clunky this felt when I was implementing it.